gulp/e2e-tests.js

```diff
@@ -10,14 +11,15 @@ var $ = require('gulp-load-plugins')();
 
 // Downloads the selenium webdriver
 gulp.task('webdriver-update', $.protractor.webdriver_update);
-
 gulp.task('webdriver-standalone', $.protractor.webdriver_standalone);
 
+gulp.task('docker-start', shell.task(['docker run -d -P -e no_proxy=localhost -e HUB_ENV_no_proxy=localhost -p 0.0.0.0:4444:4444 -p 0.0.0.0:5900:5900 --name gulp-chrome-debug selenium/standalone-chrome-debug']))
```

Comment:
Then you get an error that the command docker does not exists. :P 
The normal gulp protractor command makes no use of docker. This is only to make it easier to use for those that want to.